### ISSUE TITLE
Play Store release

### ISSUE THREAD

**Issue by denisidoro** [JHubi1/ollama-app#41]
### Requirements

- [X] I have searched the issues of this repository and believe that this is not a duplicate
- [X] I can confirm this feature request is not already implemented in the app

### Description

Could you please consider releasing it to the Play Store?

### Possible solution

Self-descriptive

### Additional context

My phone has MDM configured, and I can't sideload APKs

**Comment by DevGoran** [comment]
+1

**Comment by JHubi1** [comment]
Yes, I totally agree, that would be quite practical. The problem I'm facing is that you need a credit card to be able to publish to the PlayStore, but having a credit card is not common in my country. Debit cards are not accepted by Google. I'll look into it. Maybe something changes in the future

**Comment by DevGoran** [comment]
Alternatively, F-Droid is also quite popular, and this would also be great until maybe you've figured out how to get to a credit card.

**Comment by JHubi1** [comment]
It's already published on IzzyOnDroid, official Fdroid will most likely be released after the next release

**Comment by denisidoro** [comment]
While publishing it to alternative stores is already very nice, installing these stores is only possible by sideloading their APK, so in my particular case that isn't an option

**Comment by IanEatsBabies** [comment]
On some phones you can get a root adb connection, maybe try installing via adb root? You'll have to enable usb debugging in the developer options.
